tobac/tracking.py

```diff
@@ -113,12 +113,18 @@ def linking_trackpy(
         )
     elif method_linking == "predict":
 
+        # avoid setting pos_columns by renaimng to default values to avoid trackpy bug
+        features.rename(columns={"hdim_1": "x", "hdim_2": "y"}, inplace=True)
+
+        # generate list of features as input for df_link_iter to avoid bug in df_link
+        features_linking_list = [frame for i, frame in features.groupby("frame")]
```

Comment:
Is this guaranteed to be sorted? If not, should `features_linking_list` be sorted by `frame`?

Reply:
Yes, `sort=True` is the default for `groupby`. We don`t have to worry about this, but I think I`ll make it explicit still, just in case anyone else is wondering.

Reply:
Thanks. Pandas has shown that they are completely okay with changing defaults with little warning, so this makes me feel better.